File: eCopy.Model/Response/ApplicationUserResponse.cs
using System;

namespace eCopy.Model.Response
{
    public class ApplicationUserResponse
    {
        public int ID { get; set; }
        public string Email { get; set; }
        public string Username { get; set; }
        public string PhoneNumber { get; set; }
        public DateTimeOffset? LockoutEnd { get; set; }
        public bool PhoneNumberConfirmed { get; set; }
        public bool EmailConfirmed { get; set; }
        public bool ChangePassword { get; set; }
        public bool TwoFactorEnabled { get; set; }
        public bool Active { get; set; }
    }
}

File: ecopymobile/myapp/lib/model/registrationModel/response/applicationUserResponse.dart
import 'dart:ffi';
import 'package:json_annotation/json_annotation.dart';

part 'applicationUserResponse.g.dart';

@JsonSerializable()
class ApplicationUserResponse {
  int? id;
  String? email;
  String? username;
  String? phoneNumber;
  //DateTime? lockoutEnd;
  bool? phoneNumberConfirmed;
  bool? emailConfirmed;
  bool? changePassword;
  bool? twoFactorEnabled;
  bool? active;

  ApplicationUserResponse() {}

  factory ApplicationUserResponse.fromJson(Map<String, dynamic> json) =>
      _$ApplicationUserResponseFromJson(json);

  /// Connect the generated [_$ApplicationUserResponseToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ApplicationUserResponseToJson(this);
}
